Changelog.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Added new default `std` feature. Disabling this feature makes ash no_std capable and raises the MSRV to 1.64
```

Comment:
PR identifier? And as evident from the other entries, these are mostly chronological / ordered by PR number.